package.json

```diff
@@ -1,84 +1,84 @@
-    ],
-    "testEnvironment": "jsdom"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/guardian/mobile-apps-article-templates.git"
-  },
-  "keywords": [
-    "guardian",
-    "mobile",
-    "template"
-  ],
-  "bugs": {
-    "url": "https://github.com/guardian/mobile-apps-article-templates/issues"
-  },
-  "devDependencies": {
-    "@babel/core": "^7.2.2",
-    "@babel/preset-env": "^7.9.0",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^10.0.1",
-    "babel-jest": "^28.1.0",
-    "babel-loader": "^8.0.4",
-    "babel-plugin-dynamic-import-node": "^2.2.0",
-    "babel-plugin-syntax-dynamic-import": "^6.18.0",
-    "copy-webpack-plugin": "^11.0.0",
-    "core-js": "^3.6.5",
-    "css-loader": "^6.6.0",
-    "eslint": "^7.32.0",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-loader": "^4.0.2",
-    "eslint-plugin-import": "^2.14.0",
-    "jest": "^29.6.1",
-    "jest-canvas-mock": "^2.3.0",
-    "jshint-summary": "^0.4.0",
-    "mini-css-extract-plugin": "^2.4.5",
-    "regenerator-runtime": "^0.13.1",
-    "sass": "^1.32.8",
-    "sass-loader": "^13.0.0",
-    "style-loader": "^3.3.1",
-    "terser-webpack-plugin": "^5.3.0",
-    "webpack": "^5.65.0",
-    "webpack-cli": "^4.1.0",
-    "micromatch": "^4.0.8"
-  },
-  "dependencies": {
-    "@babel/polyfill": "^7.4.4",
-    "@changesets/cli": "^2.27.6",
-    "@guardian/fence": "^0.2.12",
-    "classlist-polyfill": "^1.2.0",
-    "curl-amd": "^0.8.12",
-    "d3": "^7.1.0",
-    "d3-shape": "^3.1.0",
-    "domready": "^1.0.7",
-    "fastclick": "^1.0.6",
-    "intersection-observer": "^0.12.2",
-    "jest-environment-jsdom": "^29.6.1",
-    "raf": "^3.4.0"
-  }
+    "bugs": {
+        "url": "https://github.com/guardian/mobile-apps-article-templates/issues"
+    },
+    "devDependencies": {
+        "@babel/core": "^7.2.2",
+        "@babel/preset-env": "^7.9.0",
+        "babel-core": "^7.0.0-bridge.0",
+        "babel-eslint": "^10.0.1",
+        "babel-jest": "^28.1.0",
+        "babel-loader": "^8.0.4",
+        "babel-plugin-dynamic-import-node": "^2.2.0",
+        "babel-plugin-syntax-dynamic-import": "^6.18.0",
+        "copy-webpack-plugin": "^11.0.0",
+        "core-js": "^3.6.5",
+        "css-loader": "^6.6.0",
+        "eslint": "^7.32.0",
+        "eslint-config-airbnb-base": "^15.0.0",
+        "eslint-loader": "^4.0.2",
+        "eslint-plugin-import": "^2.14.0",
+        "jest": "^29.6.1",
+        "jest-canvas-mock": "^2.3.0",
+        "jshint-summary": "^0.4.0",
+        "mini-css-extract-plugin": "^2.4.5",
+        "regenerator-runtime": "^0.13.1",
+        "sass": "^1.32.8",
+        "sass-loader": "^13.0.0",
+        "style-loader": "^3.3.1",
+        "terser-webpack-plugin": "^5.3.0",
+        "webpack": "^5.65.0",
+        "webpack-cli": "^4.1.0",
+        "micromatch": "^4.0.8"
+    },
+    "dependencies": {
+        "@babel/polyfill": "^7.4.4",
+        "@changesets/cli": "^2.27.10",
```

Comment:
It is the only line which was changed to bump `changesets/cli`.  The rest of the changes probably result from formatting.